src/components/tx/SpendingLimitRow/index.tsx

```diff
@@ -1,25 +1,25 @@
 import { FormControl, FormControlLabel, Radio, RadioGroup, Typography } from '@mui/material'
 import { Controller, useFormContext } from 'react-hook-form'
-import { formatVisualAmount } from '@/utils/formatters'
+import type { BigNumber } from '@ethersproject/bignumber'
+import { safeFormatUnits } from '@/utils/formatters'
 import type { TokenInfo } from '@safe-global/safe-gateway-typescript-sdk'
-import type { SpendingLimitState } from '@/store/spendingLimitsSlice'
 import { SendAssetsField, SendTxType } from '@/components/tx/modals/TokenTransferModal/SendAssetsForm'
 
 const SpendingLimitRow = ({
-  spendingLimit,
+  availableAmount,
   selectedToken,
 }: {
-  spendingLimit: SpendingLimitState
+  availableAmount: BigNumber
   selectedToken: TokenInfo | undefined
 }) => {
   const { control } = useFormContext()
 
-  const formattedAmount = formatVisualAmount(spendingLimit.amount, selectedToken?.decimals)
+  const formattedAmount = safeFormatUnits(availableAmount, selectedToken?.decimals)
```

Comment:
I decided to change this so that it doesn't cut off decimal places because it is the same as in safe-react.